### ISSUE TITLE
How to Connect Pocketbase Server to another app?

### ISSUE THREAD

**Issue by merajhasan88** [rohitsangwan01/pocketbase_server_flutter#5]
Thanks for this package. I want to connect my client app on a real device to the pocketbase server with both of them on different networks. You mention in a note on the `Readme` section of this app's `pub.dev` page that:
```
This is for running Pocketbase server from mobile, to connect with pocketbase server, use official pocketbase client plugin
```
I realize I need to expose my local address through `ngrok` but how do I integrate it? Do you have some ideas or leads?

Thanks again

**Comment by rohitsangwan01** [comment]
Yes you can expose server with ngrok or with static ip ( if your router supports it )
Then use that url in client sdk to connect
